### PR TITLE
Fix tools.jar dependency

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

File: table/server/underdb/glue/pom.xml [hunk 1]
```diff
@@ -77,5 +77,38 @@
...
+            <exclusion>
+              <groupId>org.apache.logging.log4j</groupId>
+              <artifactId>log4j-slf4j-impl</artifactId>
+            </exclusion>
+            <exclusion>
+              <groupId>com.google.protobuf</groupId>
+              <artifactId>protobuf-java</artifactId>
+            </exclusion>
+            <exclusion>
+              <groupId>jdk.tools</groupId>
```
Comment (yuzhu) [inline table/server/underdb/glue/pom.xml:106]: this is the main part of the change. the rest of the file was wrongly indented

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
@ZacBlanco you probably have more context here

File: table/server/underdb/glue/pom.xml [hunk 2]
```diff
@@ -13,69 +13,98 @@
...
-                    <artifactId>log4j-1.2-api</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.logging.log4j</groupId>
-                    <artifactId>log4j-slf4j-impl</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>com.google.protobuf</groupId>
-                    <artifactId>protobuf-java</artifactId>
-                </exclusion>
```
Comment (ZacBlanco) [inline table/server/underdb/glue/pom.xml:75]: do we not need to exclude this any more?

File: table/server/underdb/hive/pom.xml [hunk 3]
```diff
@@ -151,6 +151,10 @@
               <groupId>org.apache.logging.log4j</groupId>
               <artifactId>log4j-slf4j-impl</artifactId>
             </exclusion>
+            <exclusion>
+              <groupId>com.google.protobuf</groupId>
+              <artifactId>protobuf-java</artifactId>
+            </exclusion>
```
Comment (ZacBlanco) [inline table/server/underdb/hive/pom.xml:157]: is this added now to be consistent with the other dependency? Can we make this a dependency in table/server or table/server/underdb and share it between hive and glue so we don't have to name the exclusions list twice?

File: table/server/underdb/glue/pom.xml (hunk 2 above)
Comment (yuzhu) [reply table/server/underdb/glue/pom.xml:75]: fixed, keeping the files consistent

File: table/server/underdb/hive/pom.xml (hunk 3 above)
Comment (yuzhu) [reply table/server/underdb/hive/pom.xml:157]: my goal is to eventually remove the dependency on hive from glue. i think right now it uses some string manipulation functions from hive....

**Comment by yuzhu** [comment]
merging to unblock java 11 tests.

**Comment by yuzhu** [comment]
alluxio-bot, merge this please.

**Pull request merged**